src/SecTester.Reporter/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.Extensions.DependencyInjection;
+using SecTester.Scan.CI;
+
+namespace SecTester.Reporter.Extensions;
+
+public static class ServiceCollectionExtensions
+{
+  public static IServiceCollection AddSecTesterReporter(this IServiceCollection collection)
```

Comment:
You don't need to provide a method to register a default formatter. This is runner's responsibility.